tests/src/ut_test_kvp.c

```diff
@@ -434,6 +568,78 @@ static int test_ut_kvp_createGlobalJSONInstance( void )
     return 0;
 }
 
+static int test_ut_kvp_createGlobalYAMLInstanceForMallocedData( void )
+{
+    ut_kvp_status_t status;
+    test_ut_memory_t kvpMemory;
+
+    gpMainTestInstance = ut_kvp_createInstance();
+    if ( gpMainTestInstance == NULL )
+    {
+        assert( gpMainTestInstance != NULL );
+        UT_LOG_ERROR("ut_kvp_open() - Read Failure");
+        return -1;
+    }
+
+    if (read_file_into_memory(KVP_VALID_TEST_YAML_FILE, &kvpMemory) == 0)
```

Comment:
this is a global `gkvpMemory`, 

Which must be freed during the shutdown function.

Reply:
file moved to ut-control